src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultArray = {};
```

Comment:
The variable name 'resultArray' is misleading as it suggests that the variable is an array, while it is actually an object intended to store the styles. Consider renaming it to something more descriptive, like 'styleObject' to improve readability.